source/libraries/ContractDeployer.ts

```diff
@@ -0,0 +1,231 @@
+            saiGovContract.address,
+            saiPipContract.address,
+            saiPepContract.address,
+            saiPitAddress,
+        );
+
+        // seth send $DAI_FAB 'makeTapTop()'
+        console.log("DaiFab.makeTapTop()");
+        await daiFabContract.makeTapTop();
+
+        // seth send $DAI_FAB 'configParams()'
+        console.log("DaiFab.configParams()");
+        await daiFabContract.configParams();
+
+        console.log("DaiFab.verifyParams()");
+        // seth send $DAI_FAB 'verifyParams()'
+        await daiFabContract.verifyParams();
+
+        // seth send $DAI_FAB 'configAuth(address)' $SAI_ADM
+        console.log("DaiFab.configAuth()");
+        await daiFabContract.configAuth(saiAdmContract.address);
+
+        const deployedContractAddresses = {
+            gem: saiGemContract.address,
+            gov: saiGovContract.address,
+            pip: saiPipContract.address,
+            pep: saiPepContract.address,
+            pit: saiPitAddress,
+            adm: saiGemContract.address,
+
+            sai: await daiFabContract.sai_(),
+            sin: await daiFabContract.sin_(),
+            skr: await daiFabContract.skr_(),
+            dad: await daiFabContract.dad_(),
+            mom: await daiFabContract.mom_(),
+            vox: await daiFabContract.vox_(),
+            tub: await daiFabContract.tub_(),
+            tap: await daiFabContract.tap_(),
+            top: await daiFabContract.top_(),
+        };
+        await this.generateAddressMappingFile(deployedContractAddresses);
+    }
+
+    private async simpleDeploy(contractName: string, constructorArgs?: Array<any>): Promise<string> {
+        return await this.construct(this.contracts.get(contractName), constructorArgs || [], `Uploading ${contractName}`);
+    }
+
+    private static getEncodedConstructData(abi: Abi, bytecode: Buffer, constructorArgs: Array<string>): Buffer {
+        if (constructorArgs.length === 0) {
+            return bytecode;
+        }
+        // TODO: submit a TypeScript bug that it can't deduce the type is AbiFunction|undefined here
+        const constructorSignature = <AbiFunction | undefined>abi.find(signature => signature.type === 'constructor');
+        if (typeof constructorSignature === 'undefined') throw new Error(`ABI did not contain a constructor.`);
+        const constructorInputTypes = constructorSignature.inputs.map(x => x.type);
+        const encodedConstructorParameters = Buffer.from(encodeParams(constructorInputTypes, constructorArgs).substring(2), 'hex');
+        return Buffer.concat([bytecode, encodedConstructorParameters]);
+    }
+
+    private async construct(contract: Contract, constructorArgs: Array<string>, failureDetails: string): Promise<string> {
+        const data = `0x${ContractDeployer.getEncodedConstructData(contract.abi, contract.bytecode, constructorArgs).toString('hex')}`;
+        // TODO: remove `gas` property once https://github.com/ethereumjs/testrpc/issues/411 is fixed
```

Comment:
This has been fixed, can be addressed.

Reply:
Not only has it been fixed, it has been addressed in this file, if i understand issue, but the comment wasn't removed?

https://github.com/AugurProject/augur-core/commit/4a6f9ddd108a357e19bd14c969cd2298657e3a8b#diff-1896c05bf64c6298f97512471d4f123aL150